qiskit_dynamics/pulse/__init__.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+==============================================
+Pulse (:mod:`qiskit_dynamics.pulse`)
+==============================================
+
+.. currentmodule:: qiskit_dynamics.pulse
+
+This module contains converters that convert Qiskit pulse schedules into signals such that
```

Comment:
Since we might be putting more stuff into this module it might be worth making this first sentence a bit more general, like:

"This module contains tools for interfacing qiskit pulse with qiskit dynamics."

Then have a section called "Converters" that talks more specifically about schedule -> Signal conversion. The general statement about the need for schedule -> Signal conversion in `InstructionToSignals` (mentioned below) I think would be better here than in the class doc string.

Reply:
See here: 094e9f2